### ISSUE TITLE
List search results hidden in Form mode

### ISSUE THREAD

**Issue by TopCat6** [DNNCommunity/DNN.FormAndList#14]
FnL 6.2.0 on DNN 8.0.4 with Form only, showing 'Show Records' button, and click it to see records. Then do a search and the results are not displayed; only the form again. Editing the URL in the browser at that point to add '/show/records' shows the proper search results. It seems to happen only when using the XSLT option, and doesn't happen with the Auto option.

BTW, the paging buttons properly include the '/show/records?' code in the URL. 

'Form above List' etc. modes do not have this issue, by their nature since they always show the list.

**Comment by valadas** [comment]
@DNNCommunity/module-dev-team I can reproduce this issue in the latest build. However adding a search field in the xslt is marked as obsolete in the UI, any idea of the reasoning to mark this obsolete? It was already like that from CodePlex so there is no easy way to find how that got in there and why.

**Comment by SCullman** [comment]
@valadas:
Good question. It was years ago when I marked it as obsolete as a mark to myself that I am not going to extend that feature. 

Basically, it works. But on the other hand it is done within XSLTand it is not really aware of the current schema:
```
    <xsl:variable name="currentData" select="udt:Data[contains(*[name()=$searchColumns][1],$search) or contains(*[name()=$searchColumns][2],$search) or contains(*[name()=$searchColumns][3],$search) or contains(*[name()=$searchColumns][4],$search) or contains(*[name()=$searchColumns][5],$search)]" />
```

Good enough for me and others with enough knowledge of XSLT, but a never ending source of pain for ordinary users. I wanted to discourage people using it, therefor I claimed it as obsolete. 

However, this has nothing to do with this issue.
